src/components/Display.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return (
+    <div>
+      <input type="text" name="result" id="resultOPeration" value={result} />
```

Comment:
A input field, I dont think it is perfect for this scenario. You can add a p tag instead of it. Cause the disply should render the dault and I should not be able to click it to see a cursor. Is't it?